internal/worker/apiremotecaller/remote.go

```diff
@@ -212,8 +239,7 @@ func (w *remoteServer) loop() error {
 				continue
 			}
 
-			var err error
-			monitor, err = w.connect(rctx, addresses)
+			conn, err := w.connect(rctx, addresses)
```

Comment:
Do we need specific error handling for when `rctx` is cancelled vs `w.tomb` dying?

Reply:
I don't think so, it's all the same really.